R/expr__expr.R

```diff
@@ -4252,13 +4252,32 @@ Expr_shrink_dtype = "use_extendr_wrapper"
 
 
 
-#' arr: list related methods
+#' arr: list related methods DEPRECATED
+#' @description
+#' DEPRECATED FROM 0.9.0 USE `<Expr>$list$...` instead. Subnamespace is simple renamed.
+#' @keywords Expr
+#' @return Expr
+#' @aliases arr_ns
+#' @seealso \code{\link[=Expr_list]{<Expr>$list$...}}
+Expr_arr = method_as_property(function() {
+  if (!isTRUE(runtime_state$warned_deprecate_sns_arr)) {
+    warning(
+      "in <Expr>$list$: `<Expr>$list$...` is deprecated and removed from polars 0.9.0 . ",
```

Comment:
should be "<Expr>$arr" instead of "<Expr>$list"

Same comment as above about version